airbyte-webapp/src/packages/cloud/views/workspaces/WorkspaceSettingsView/WorkspaceSettingsView.tsx

```diff
@@ -27,13 +28,18 @@ const AdvancedModeSwitchLabel = () => (
   </>
 );
 
+const ValidationSchema = yup.object().shape({
+  name: yup.string().required("form.empty.error"),
+});
+
 export const WorkspaceSettingsView: React.FC = () => {
   const { formatMessage } = useIntl();
   useTrackPage(PageTrackingCodes.SETTINGS_WORKSPACE);
   const { exitWorkspace } = useWorkspaceService();
   const workspace = useCurrentWorkspace();
-  const removeWorkspace = useRemoveWorkspace();
-  const updateWorkspace = useUpdateWorkspace();
+  const removeCloudWorkspace = useRemoveCloudWorkspace();
+  const updateCloudWorkspace = useUpdateCloudWorkspace();
```

Comment:
I think typically we do something like:
```suggestion
  const { mutateAsync: updateCloudWorkspace } = useUpdateCloudWorkspace();
```